examples/wagmi/src/App.tsx

```diff
@@ -0,0 +1,175 @@
+import { useEffect, useState } from 'react';
+import { useAccount, useConnect, useDisconnect, usePublicClient,
+         useWaitForTransactionReceipt, useWalletClient } from 'wagmi'
+import { writeContract } from 'viem/actions';
+
+const StorageABI = [
+  {
+    "inputs": [],
+    "name": "retrieve",
+    "outputs": [
+      {
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "uint256",
+        "name": "num",
+        "type": "uint256"
+      }
+    ],
+    "name": "store",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  }
+] as const;
+
+function App() {
+  const account = useAccount();
+  const { connectors, connect, status, error } = useConnect();
+  const { disconnect } = useDisconnect();
+  const { data: walletClient } =  useWalletClient();
+  const [deployHash, setDeployHash] = useState<undefined | `0x${string}`>();
+  const [contractAddress, setContractAddress] = useState<undefined | `0x${string}`>();
+  const [writeTxHash, setWriteTxHash] = useState<undefined | `0x${string}`>();
+  const [readResult, setReadResult] = useState<bigint|undefined>();
+  const publicClient = usePublicClient();
+  const {
+    data:deployReceipt,
+    error:deployTxError,
```

Comment:
Do we have `prettier` in this repo? This doesn't looks correctly formatted to me. I would expect a space after `:`

Reply:
I'm using `biome`, have re-formatted using it.